### PR TITLE
Fancynancy

### PR DESCRIPTION
**Pull request by berrybus**
Hi, I added some basic tests to part 5 with the RegisterServlet. The tests aren't very complicated because I'm a little confused by how the test cases work, so maybe Mahmoud can help with that. In addition, I started on part 9, the password hashing. I'm missing a line because it relies on stuff from parts 6/7, but once that's in, I'll add it. It's basically just making sure that you actually have to enter in the passwords.

### CODE REVIEW

**Review by luisgruba — APPROVED** [review]
Looks good! Once 6/7 comes in, I'm sure the rest will work. I'll approve this pull request, but in the meantime, lets not merge just yet.

File: src/main/java/codeu/controller/RegisterServlet.java [hunk 1]
```diff
@@ -82,7 +85,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
    //End check and add user if valid. Redirects to login page
 
-   User user = new User(UUID.randomUUID(), username, password, Instant.now());
```
Comment (halikins) [inline src/main/java/codeu/controller/RegisterServlet.java:85]: This was a part of Step 1 and Step 2 and later steps made changes to name

File: src/main/java/codeu/model/store/basic/DefaultDataStore.java [hunk 2]
```diff
@@ -99,7 +100,7 @@ private void addRandomUsers() {
 
     for (int i = 0; i < DEFAULT_USER_COUNT; i++) {
       //PART 1: Temp "password" passed to allow proper compilation 
-      User user = new User(UUID.randomUUID(), randomUsernames.get(i), "password" ,Instant.now());
```
Comment (halikins) [inline src/main/java/codeu/model/store/basic/DefaultDataStore.java]: This was a part of Step 1 and Step 2 and later steps made changes to name

Comment (halikins) [inline src/main/java/codeu/model/store/basic/DefaultDataStore.java:102]: Spacing issue?

Comment (berrybus) [reply src/main/java/codeu/model/store/basic/DefaultDataStore.java:102]: Oh yeah, oops, I'll fix that

Comment (luisgruba) [reply src/main/java/codeu/model/store/basic/DefaultDataStore.java:102]: The passwords from part 1 and 2 were changed to password Hashes in part 8/9

Comment (halikins) [reply src/main/java/codeu/model/store/basic/DefaultDataStore.java:102]: Yeah. I noticed that as well.

**Pull request merged**